homeworks/Aleksandr.Turuta_turuta7/homework_3/function.js

```diff
@@ -0,0 +1,24 @@
+function sq(n) {
+    const x = n ** 2;
```

Comment:
avoid using `**` operator here

Reply:
Is it better to use  Math.pow?

Reply:
the condition is not to use built in Math methods and exponential operators

Reply:
for example yes?

let num = 50;
let n = 3;

function test (n){
    let p = num;
     for (let i=0;i<n-1;i++) {
        p *= num;}    
        return(p);
    }
    
let zzz = test(n); 
    console.log(zzz);

Reply:
by the simple?

Reply:
in this case you could just `return n * n`

Reply:
in this example, yes.